lib/active_model/serializer/fieldset.rb

```diff
@@ -20,11 +20,7 @@ def fields_for(type)
       private
 
       def parsed_fields
-        if raw_fields.is_a?(Hash)
-          raw_fields.each_with_object({}) { |(k, v), h| h[k.to_sym] = v.map(&:to_sym) }
-        else
-          {}
-        end
+        JSONAPI::IncludeDirective.new(raw_fields).to_hash
```

Comment:
or just that `fields_for` should return `{}`, n/m  https://github.com/rails-api/active_model_serializers/blob/91b37cee2c0815234d94214219497c0a26d33ffd/lib/active_model/serializer/fieldset.rb#L12-L14


```diff
      def fields_for(type)
-        fields[type.singularize.to_sym] || fields[type.pluralize.to_sym]
+        fields[type.singularize.to_sym] || fields[type.pluralize.to_sym] || {}
      end
```